File: FMAplication/Domain/DailyTasks/DailyConsumerSurveyTaskAnswer.cs
using FMAplication.Core;
using FMAplication.Domain.Questions;

namespace FMAplication.Domain.DailyTasks
{
    public class DailyConsumerSurveyTaskAnswer : AuditableEntity<int>
    {
        public DailyConsumerSurveyTask DailyConsumerSurveyTask { get; set; }
        public int DailyConsumerSurveyTaskId { get; set; }

        public Question Question { get; set; }
        public int QuestionId { get; set; }
        public string Answer { get; set; }
    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/DailyTasks/dailyConsumerSurveyTaskAnswer.ts
import { Question } from "..";

export class DailyConsumerSurveyTaskAnswer{
    id: number;
    createdBy: number;
    createdTime: Date;
    createdTimeStr: string;
    modifiedBy: number;
    modifiedTime: Date;
    modifiedTimeStr: string;
    status: number;
    dailyConsumerSurveyTaskId: number;
    question: Question;
    questionId: number;
    answer: string;    
}